src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressWithPlainTextCommentFilter.java

```diff
@@ -588,6 +578,46 @@ private boolean isInScopeOfSuppression(AuditEvent event) {
             return lineNo <= event.getLine();
         }
 
+        /**
+         * Checks whether {@link AuditEvent} source name matches the check format.
+         * @param event {@link AuditEvent} instance.
+         * @return true if the {@link AuditEvent} source name matches the check format.
+         */
+        private boolean isCheckMatch(AuditEvent event) {
+            final Matcher checkMatcher = eventSourceRegexp.matcher(event.getSourceName());
+            return checkMatcher.find();
+        }
+
+        /**
+         * Checks whether the {@link AuditEvent} module ID matches the ID format.
+         * @param event {@link AuditEvent} instance.
+         * @return true if the {@link AuditEvent} module ID matches the ID format.
+         */
+        private boolean isIdMatch(AuditEvent event) {
+            boolean match = false;
+            if (event.getModuleId() != null) {
+                final Matcher idMatcher = eventSourceRegexp.matcher(event.getModuleId());
+                match = idMatcher.find();
+            }
+            return match;
+        }
+
+        /**
+         * Checks whether the {@link AuditEvent} message matches the message format.
+         * @param event {@link AuditEvent} instance.
+         * @return true if the {@link AuditEvent} message matches the message format.
+         */
+        private boolean isMessageMatch(AuditEvent event) {
+            final boolean match;
+            if (eventMessageRegexp == null) {
+                match = true;
+            }
```

Comment:
Please make it like `boolean match = true;` and test `eventMessageRegexp != null`

Reply:
Ah, good call - I should have done it that way in the other two filters as well (SuppressionComment and SuppressWithNearbyComment) - I’d be happy to update those too if you’d like, if a minor change like that to align all 3 makes sense in this PR. But I understand if better to leave those alone in this PR.

Reply:
No, let's do simple refactoring (no functional changes) for others in separate PR in commit message as "minor:"